src/Components/RingdownCard.scss

```diff
@@ -29,7 +29,19 @@
   &.usa-card__container {
     background-color: color($theme-color-success-lighter);
   }
-  .ringdown-card__eta {
+  .ringdown-card__status {
     color: color($theme-color-base-ink);
   }
 }
+
+.ringdown-card--cancelled {
+  &.usa-card__container {
+    background-color: color($theme-color-warning);
+  }
+  .ringdown-card__status {
```

Comment:
And here I override .ringdown-card__status when in cancelled state, changing the text color and content alignment.

In doing it like this, we don't need another new separate .ringdown-cancel style...